portfolio/src/main/java/com/google/sps/data/ScheduledTask.java

```diff
@@ -24,4 +28,13 @@ public Task getTask() {
   public Instant getStartTime() {
     return startTime;
   }
+
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof ScheduledTask && equals(this, (ScheduledTask) other);
+  }
+
+  public static boolean equals(ScheduledTask a, ScheduledTask b) {
```

Comment:
Is this method necessary? Each time we call ScheduledTask.equals(a, b), we can just call a.equals(b).

Reply:
This code came almost directly from a week 5 walk-through. This method is mainly to simplify the logic for the other .equals(). If I remove this then instead of `&& equals(this, (ScheduledTask) other)` in the first .equals(), I would need something like:
```
return other instanceof ScheduledTask
        && this.task.equals(((ScheduledTask) b).task)
        && this.startTime.equals(((ScheduledTask) b).startTime);
```
Do you recommend I leave this method in or remove it?

Reply:
You can leave it here if you want.